com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/EmberNcp.java

```diff
@@ -175,31 +173,17 @@ public class EmberNcp {
     private final Logger logger = LoggerFactory.getLogger(EmberNcp.class);
 
     /**
-     * The protocol handler used to send and receive EZSP packets
+     * The ZigBee dongle used to send and receive EZSP packets
      */
-    private final EzspProtocolHandler protocolHandler;
-
-    /**
-     * The status value from the last request
-     */
-    private EmberStatus lastStatus;
+    private final ZigBeeDongleEzsp zigBeeDongleEzsp;
 
     /**
      * Create the NCP instance
      *
-     * @param protocolHandler the {@link EzspFrameHandler} used for communicating with the NCP
-     */
-    public EmberNcp(EzspProtocolHandler protocolHandler) {
-        this.protocolHandler = protocolHandler;
-    }
-
-    /**
-     * Returns the {@link EmberStatus} from the last request. If the request did not provide a status, null is returned.
-     *
-     * @return {@link EmberStatus}
+     * @param zigBeeDongleEzsp the {@link ZigBeeDongleEzsp} used for communicating with the NCP
      */
-    public EmberStatus getLastStatus() {
```

Comment:
Why has this method been removed? This is quite a large breaking change.

Reply:
Same as comment above.